app/controllers/admin/custom_organisations_controller.rb

```diff
@@ -30,4 +32,24 @@ def destroy
   def custom_org_params
     params.require(:custom_organisations).permit(:name)
   end
+
+  def order_custom_orgs
```

Comment:
now that this is going to be returning the sorted names, this can be changed to `ordered_custom_orgs`.

Also no longer needs to set the variable